packages/currency/src/erc20/index.ts

```diff
@@ -17,6 +17,7 @@ export function getSupportedERC20Tokens(): ERC20Currency[] {
           network: networkName,
           decimals: token.decimals,
           symbol: token.symbol,
+          id: token.id,
```

Comment:
@KolevDarko The tokens do not have an `id` (see the [JSON file](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/currency/src/erc20/chains/sepolia.ts)), you can create it manually maybe.
```suggestion
          id: `${token.symbol}-${networkName}`,
```